mathesar/views/__init__.py

```diff
@@ -22,6 +22,19 @@
 __all__ = [export, users, download_link]
 
 
+import time
+from django.http import JsonResponse
+
+from django.views.decorators.csrf import csrf_exempt
+@csrf_exempt
+def slow_view(request):
+    from mathesar.rpc.explorations import run_saved
+    t1 = time.perf_counter()
+    exp = run_saved(exploration_id=6, request=request)
```

Comment:
Change the `exploration_id` to your big exploration that takes 2-3 secs to load(for testing using locust)